test/orb.jl

```diff
@@ -33,7 +33,7 @@ facts("ORB") do
         desc_2, ret_keypoints_2 = create_descriptor(img_array_2, orb_params)
         matches = match_keypoints(ret_keypoints_1, ret_keypoints_2, desc_1, desc_2, 0.2)
         reverse_keypoints_1 = [_reverserotate(m[1], pi / 4, (256, 384)) for m in matches]
-        @fact all(isapprox(rk[1], m[2][1], atol = 3) && isapprox(rk[2], m[2][2], atol = 3) for (rk, m) in zip(reverse_keypoints_1, matches)) --> true
+        @fact (grade_matches(reverse_keypoints_1, [match[2] for match in matches], (i,j)->(abs(i[1]-j[1]) + abs(i[2]-j[2]))) < 1.5 ) --> true
```

Comment:
I was of the opinion that the other version is redundant 😅 My reasons for the `limit` version were (I wanted it to be something like the `isapprox` function)- 

- The other version just returns the mean `diff` over two sets of keypoints. The `mean` could be large if just some outlier pairs of matched keypoints have a very large distance (could be possible that two completely wrong keypoints get matched)
- How do we define passing tests? IMO these would be those pairs which have distance within some `limit`. Also, the mean does not tell us what *proportion* of the keypoints are well matched. 
- The `mean` can mean (pun not intended 😛) different things for differently sized images. For a very large image, a `mean` distance of 15 pixels might mean that the keypoints are quite well matched, whereas for a small one it means they are poorly matched. Instead, if we just pass the `limit` (which can be passed as 1% of the width for example) and get the percentage of keypoints within this limit, it gives a much better way to judge the quality of keypoints obtained without needing to specify the exact distance each time.
- We could return the number of keypoints under the `limit` instead of percentage but that might lead to wrong conclusions (100 keypoints passed looks good but that depends if the total was 110 or 1000).

I might be completely wrong about this though. Does this make sense?